### PR TITLE
Docstring for import_from_path method

### PR DESCRIPTION
**Pull request by ibrsam**
Descriptions of docstrings for import_from_path and import_single_calculation methods.

### CODE REVIEW

File: pyiron/project.py [hunk 1]
```diff
@@ -258,12 +258,13 @@ def import_single_calculation(
         self, project_to_import_from, rel_path=None, job_type="Vasp", copy_raw_files=False
     ):
         """
-
+        A method to import a single-calculation jobs into pyiron. Currently, it suppor
```
Comment (pmrv) [inline pyiron/project.py:261]: The linebreak shouldn't be inside a word.

**Review by pmrv — APPROVED** [review]
Otherwise looks great, thanks a lot!

**Comment by samwaseda** [comment]
Wait shouldn't you have fixed the codacy complaints?

**Comment by pmrv** [comment]
Ah, I missed those. Fixed directly on master. 595bbc0a53ab1e841e826142e4fcbc2e3bc6691d

**Pull request merged**